InventorSatExport/InventorSatExportPlugin/SampleAutomation.cs

```diff
@@ -0,0 +1,160 @@
+﻿/////////////////////////////////////////////////////////////////////
+// Copyright (c) Autodesk, Inc. All rights reserved
+// Written by Forge Partner Development
+//
+// Permission to use, copy, modify, and distribute this software in
+// object code form for any purpose and without fee is hereby granted,
+// provided that the above copyright notice appears in all copies and
+// that both that copyright notice and the limited warranty and
+// restricted rights notice below appear in all supporting
+// documentation.
+//
+// AUTODESK PROVIDES THIS PROGRAM "AS IS" AND WITH ALL FAULTS.
+// AUTODESK SPECIFICALLY DISCLAIMS ANY IMPLIED WARRANTY OF
+// MERCHANTABILITY OR FITNESS FOR A PARTICULAR USE.  AUTODESK, INC.
+// DOES NOT WARRANT THAT THE OPERATION OF THE PROGRAM WILL BE
+// UNINTERRUPTED OR ERROR FREE.
+/////////////////////////////////////////////////////////////////////
+
+using System;
+using System.Diagnostics;
+using System.Runtime.InteropServices;
+
+using Inventor;
+using Autodesk.Forge.DesignAutomation.Inventor.Utils;
+using System.Text;
+
+namespace InventorSatExportPlugin
+{
+    [ComVisible(true)]
+    public class SampleAutomation
+    {
+        private readonly InventorServer inventorApplication;
+
+        public SampleAutomation(InventorServer inventorApp)
+        {
+            inventorApplication = inventorApp;
+        }
+
+        public void Run(Document doc)
+        {
+            LogTrace("Start Processing Run method");
+            NameValueMap map = inventorApplication.TransientObjects.CreateNameValueMap();
+            RunWithArguments(doc, map);
+        }
+
+        public void RunWithArguments(Document doc, NameValueMap map)
+        {
+            LogTrace("Processing " + doc.FullFileName);
+
+            try
+            {
+                //string dirPath = System.IO.Path.GetDirectoryName(doc.FullDocumentName);
+                string currentDirPath = System.IO.Directory.GetCurrentDirectory();
+
+                #region ExportSAT file 
+
+                Trace.TraceInformation("Export SAT file.");
+                TranslatorAddIn oSAT = null;
+
+                foreach (ApplicationAddIn item in inventorApplication.ApplicationAddIns)
+                {
+
+                    if (item.ClassIdString == "{89162634-02B6-11D5-8E80-0010B541CD80}")
+                    {
+                        Trace.TraceInformation("Finded the PDF addin.");
```

Comment:
The trace information has a minor issue, the PDF add-in is not found :)

Reply:
The log message should read "Found the PDF addin" not finded.